cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/ResolveMavenDependenciesTask.java

```diff
@@ -60,16 +77,15 @@ public void execute(BuildContext buildContext) {
         if (mavenDependencies.size() > 0) {
             buildContext.out().println("Resolving Maven dependencies\n\tDownloading dependencies into " + targetRepo);
             for (Library library : mavenDependencies) {
-                if (library.getPath() == null) {
-                    try {
-                        Dependency dependency = resolver.resolve(library.getGroupId(), library.getArtifactId(),
-                                library.getVersion(), false);
-                        library.setPath(Utils.getJarPath(targetRepo, dependency));
-                    } catch (MavenResolverException e) {
-                        buildContext.err().print("cannot resolve " + library.getArtifactId());
-                    }
+                try {
+                    Dependency dependency = resolver.resolve(library.getGroupId(), library.getArtifactId(),
+                            library.getVersion(), false);
+                    library.setPath(Utils.getJarPath(targetRepo, dependency));
+                } catch (MavenResolverException e) {
+                    buildContext.err().print("cannot resolve " + library.getArtifactId());
```

Comment:
Let's append the exception value (`e`) also to the error here.

Reply:
Also shouldn't we be throwing a LauncherException if there is an exception? Are we continuing even if the dependency resolution fails?

Reply:
We should terminate the build if dependency resolution fails. Fixed in 8ceca3732e6b9822ea74be97ea440c34042aba2d